public/offline.html

```diff
@@ -92,7 +97,7 @@
     }
 
     .signature {
-      margin-top: 40px;
+      margin-top: 20px;
```

Comment:
I adjusted these margins to prevent the dropdown selector from covering up the signature.